tmux.conf

```diff
@@ -61,14 +61,9 @@ if-shell "[ -f ~/.tmux.conf.local ]" 'source ~/.tmux.conf.local'
 unbind ^T
 bind ^T split-window -h -p 20
 
-# Scroll with mouse
-setw -g mode-mouse on
-
-# Allow mouse to select which pane to use
-set -g mouse-select-pane on
-
-# Allow mouse dragging to resize panes
-set -g mouse-resize-pane on
-
-# Allow mouse to select windows
-set -g mouse-select-window on
```

Comment:
Looks like we're getting rid of resizing panes and selecting panes/windows. We just wanted to fix the scrolling, right?